src/containers/Application/Dashboard.js

```diff
@@ -30,28 +40,33 @@ const ApplicationDashboardContainer = () => {
     return unsubscribe
   }, [setRelevantDates])
 
-  const hackerStatusObject = application.status
-  let hackerStatus =
-    hackerStatusObject !== undefined &&
-    (hackerStatusObject.applicationStatus === 'accepted'
-      ? hackerStatusObject.responded
-        ? hackerStatusObject.attending
-          ? 'acceptedAndAttending'
-          : 'acceptedNotAttending'
-        : 'acceptedNoResponseYet'
-      : hackerStatusObject.applicationStatus)
+  const { applicationStatus, responded, attending } = application.status
+  let hackerStatus
 
-  // handle case where applicationStatus: scored
-  hackerStatus =
-    hackerStatus === APPLICATION_STATUS.scored ? APPLICATION_STATUS.applied : hackerStatus
+  if (applicationStatus === 'accepted') {
+    if (responded) {
+      if (attending) {
+        hackerStatus = 'acceptedAndAttending'
+      } else {
+        hackerStatus = 'acceptedUnRSVP'
+      }
+    } else if (isRsvpOpen) {
+      hackerStatus = 'acceptedNoResponseYet'
+    } else {
+      hackerStatus = 'acceptedNoRSVP'
+    }
+  } else if (applicationStatus === 'scored') {
+    hackerStatus = 'applied'
+  } else {
+    hackerStatus = applicationStatus
+  }
 
-  const canRSVP =
-    hackerStatus === 'acceptedNoResponseYet' || hackerStatus === 'acceptedNotAttending'
-  const setRSVP = canRSVP => {
+  const canRSVP = hackerStatus === 'acceptedNoResponseYet' || hackerStatus === 'acceptedUnRSVP'
```

Comment:
It seems like the RSVP button is not displayed on the No RSVP dashboard - even though they technically missed their chance to RSVP, based on the designs, I think we should still display a disabled RSVP button. Can you modify this? 
```suggestion
  const canRSVP = hackerStatus === 'acceptedNoResponseYet' || hackerStatus === 'acceptedUnRSVP' || hackerStatus === 'acceptedNoRSVP'
```
<img width="1610" alt="Screen Shot 2020-12-31 at 5 18 29 PM" src="https://user-images.githubusercontent.com/38872354/103431914-c0f94b80-4b8c-11eb-99cc-8e80093e8ec1.png">
<img width="1563" alt="Screen Shot 2020-12-31 at 5 19 33 PM" src="https://user-images.githubusercontent.com/38872354/103431915-c22a7880-4b8c-11eb-83fd-30481f7a223e.png">